cfy_manager/main.py

```diff
@@ -100,6 +100,16 @@ def _load_config_and_logger(verbose=False,
         manager_config[SECURITY][ADMIN_PASSWORD] = admin_password
 
 
+def _print_finish_message():
+    config.load_config()
```

Comment:
You don't need to load config again - it was loaded once at the beginning of `install`, and that's enough.